doc/rados/deployment/ceph-deploy-mds.rst

```diff
@@ -44,3 +73,15 @@ Coming soon...
 
 
 .. _MDS Config Reference: ../../../cephfs/mds-config-ref
+
+Remove a Metadata Server [Manualy]
+========================
+
+1. Remove mds section in your config file ::
+
+	[mds.a]
+	host = {hostname}
+	
+2. Remove mds from cluster ::
+
+	$ sudo ceph mds rm mds.a
```

Comment:
These instructions are likely to not work as-is on a real system.  "mds rm" can only be run on inactive MDSs, so this needs to explain about stopping the daemon, then running "mds fail" on the gid.  Actually, at that point I'm not sure you need to run "mds rm" at all -- try it out?